tests/histogram.rs

```diff
@@ -560,3 +560,10 @@ fn subtract_underflow_guarded_by_per_value_count_check() {
         h.subtract(h2).unwrap_err()
     );
 }
+
+#[test]
+fn recorded_only_zeros() {
+    let mut h = Histogram::<u64>::new(1).unwrap();
+    h += 0;
+    assert_eq!(h.iter_recorded().collect::<Vec<_>>().len(), 1);
```

Comment:
```suggestion
    assert_eq!(h.iter_recorded().count(), 1);
```

Reply:
No, this is incorrect, the bug was that the iterator had the wrong number of values, not that the count was incorrect.

Reply:
This simplification does not test the fix as `.count()` was correct before this PR, but the iteration was skipping items. See #130.